### PR TITLE
d3d12: Fix for #1459

### PR DESCRIPTION
**Pull request by vlj**

### CODE REVIEW

**Comment by danilaml** [comment]
@vlj any response from nvidia devs, before we merge workaround?

**Comment by tambry** [comment]
@danilaml Would be also nice to ask AMD devs.

**Comment by vlj** [comment]
Actually the problem is a long standing one :
http://www.gamedev.net/topic/663243-slow-shader-compile-time-when-using-large-arrays/

**Comment by tambry** [comment]
But has Nvidia actually ever been contacted or asked about the problem?
And would it be also possible to contact AMD regarding the thing failing on their drivers?

**Comment by vlj** [comment]
no I don't really know how to contact them tbh.

**Comment by vlj** [comment]
Use "Buffer<float4>" instead of tbuffer which fits every drivers.

File: rpcs3/config.h [hunk 1]
```diff
@@ -862,6 +862,8 @@ namespace rpcs3
 				entry<u32>  adaptater           { this, "D3D Adaptater",       1 };
 				entry<bool> debug_output        { this, "Debug Output",        false };
 				entry<bool> overlay             { this, "Debug overlay",       false };
+				entry<bool> vertex_attribute_texture_1d
```
Comment (tambry) [inline rpcs3/config.h]: Leftover.

**Comment by vlj** [comment]
Removed left over config entry.

**Comment by danilaml** [comment]
@vlj PCSX2 example here: https://community.amd.com/thread/194895

**Comment by vlj** [comment]
@danilaml thanks for the link.
However I'm not sure AMD driver is wrong there since Microsoft Basic Renderer (which I assume is a reference rasteriser) fails to render anything too.

**Comment by danilaml** [comment]
@vlj don't know the best way to contact nvidia but this: https://devtalk.nvidia.com/ seems like an option.

**Comment by danilaml** [comment]
So is this correct workaround (that doesn't have any drawbacks)? I still think you should report it in some way.

**Pull request merged**